dask_cuda/__init__.py

```diff
@@ -48,3 +50,17 @@
     dask.dataframe.shuffle.shuffle_group
 )
 dask.dataframe.core._concat = unproxify_decorator(dask.dataframe.core._concat)
+
+
+def _register_cudf_spill_aware():
+    import cudf
+
+    # Only enable Dask/cuDF spilling if cuDF spilling is disabled
+    if not cudf.get_option("spill"):
+        from cudf.comm import serialize
+
+
```

Comment:
It would be good with a comment here saying why this it needed and include a link to https://github.com/rapidsai/dask-cuda/issues/1363

Reply:
Done via https://github.com/rapidsai/dask-cuda/pull/1369/commits/29751de83cb081a357d628136c7beaac2b9b9805